keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -1,53 +1,20 @@
 package com.netflix.spinnaker.keel.api.artifacts
 
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.BRANCH_JOB_COMMIT_BY_JOB
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.SEMVER_JOB_COMMIT_BY_JOB
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.SEMVER_JOB_COMMIT_BY_SEMVER
-import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy.SEMVER_TAG
-
 enum class ArtifactType {
-  deb, docker
+  deb, docker;
+
+  override fun toString() = name
 }
 
 enum class ArtifactStatus {
   FINAL, CANDIDATE, SNAPSHOT, RELEASE, UNKNOWN
 }
 
-sealed class DeliveryArtifact {
+abstract class DeliveryArtifact {
   abstract val name: String
   abstract val type: ArtifactType
   abstract val versioningStrategy: VersioningStrategy
   abstract val reference: String // friendly reference to use within a delivery config
   abstract val deliveryConfigName: String? // the delivery config this artifact is a part of
-  override fun toString() = "$type:$name" + if (reference != null) " (ref: $reference)" else ""
-}
-
-data class DebianArtifact(
-  override val name: String,
-  override val deliveryConfigName: String? = null,
-  override val reference: String = name,
-  val vmOptions: VirtualMachineOptions,
-  val statuses: Set<ArtifactStatus> = emptySet(),
-  override val versioningStrategy: VersioningStrategy = DebianSemVerVersioningStrategy
-) : DeliveryArtifact() {
-  override val type = ArtifactType.deb
-}
-
-data class DockerArtifact(
```

Comment:
Moved to its own class outside the api package so it can later be moved out of keel-api into another module.